package.json

```diff
@@ -128,10 +140,12 @@
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-proposal-class-properties": "^7.0.0",
     "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-proposal-private-property-in-object": "7.21.0",
```

Comment:
It would be nice to add this package to package.json of `jest-config-stripes`

Reply:
Zak created a ticket to rename some babel plugins that have become a standard - [STRWEB-87](STRWEB-87)
It looks like this rename caused a breaking change in babel/preset-env dependencies